src/Ookii.Dialogs.WinForms/InputDialogForm.cs

```diff
@@ -67,6 +67,36 @@ public bool UsePasswordMasking
             set { _inputTextBox.UseSystemPasswordChar = value; }
         }
 
+        public bool MultiLine
+        {
+            get { return _inputTextBox.Multiline; }
+            set {
+                if (value != _inputTextBox.Multiline)
+                {
+                    _inputTextBox.Multiline = value;
+
+                    if(value == true)
+                    {
+                        AcceptButton = null;
+
+                        int offset = _inputTextBox.Height * 2;
+                        _inputTextBox.Top -= offset;
+                        _inputTextBox.Height += offset;
+                        Height += offset;                        
```

Comment:
Similar to the comment above on early returns, let's add a `return` here immediately after updating the `Height` of the form and that way we can get rid of the `else` as well (and reduce nesting